docs/workflows.md

```diff
@@ -46,66 +46,51 @@ Enter a version number in the format `v1.2.3`, then hit the green `Run workflow`
 Otherwise, you will have to fill them in manually and push a commit.
 
 <details>
-<summary>Environment variables section of the release workflow</summary>
-
-```yaml
-env:
-  # The base filename of the binary produced by `cargo build`.
-  BINARY: bevy_template
-  # The name to use for the packaged application produced by this workflow.
-  PACKAGE_NAME: bevy-template
-  # The itch.io page to upload to, in the format: `user-name/project-name`.
-  # Comment this out to disable.
-  ITCH_TARGET: the-bevy-flock/bevy-template
-  # The organization or author that owns the rights to the game.
-  OWNER: the-bevy-flock
-  # The path to the assets directory.
-  ASSETS_DIR: assets
-  # Whether packages produced by this workflow should be uploaded to the Github release.
-  UPLOAD_PACKAGES_TO_GITHUB_RELEASE: true
-  # Before enabling LFS, please take a look at GitHub's documentation for costs and quota limits:
-  # https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage
-  USE_GIT_LFS: false
-```
-
+  <summary>Environment variables section of the release workflow</summary>
+
+  ```yaml
+  env:
+    # The base filename of the binary produced by `cargo build`.
+    BINARY: bevy_template
+    # The name to use for the packaged application produced by this workflow.
+    PACKAGE_NAME: bevy-template
+    # The itch.io page to upload to, in the format: `user-name/project-name`.
+    # Comment this out to disable.
+    ITCH_TARGET: the-bevy-flock/bevy-template
+    # The organization or author that owns the rights to the game.
+    OWNER: the-bevy-flock
+    # The path to the assets directory.
+    ASSETS_DIR: assets
+    # Whether packages produced by this workflow should be uploaded to the Github release.
+    UPLOAD_PACKAGES_TO_GITHUB_RELEASE: true
+    # Before enabling LFS, please take a look at GitHub's documentation for costs and quota limits:
+    # https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage
+    USE_GIT_LFS: false
+  ```
 </details>
 
-In any case, make note of the value used for `ITCH_TARGET`, as you will need it later.
```

Comment:
I'd leave this.

Reply:
I could leave it in, but I'd be surprised if a user got tripped up due to the absence of this line since it's already explained at the step where it's actually used.

Reply:
aight, I'll trust you